src/book/02-system/01-foundations/03-continuous-integration.mdx

```diff
@@ -135,3 +157,35 @@ For more information about agent configuration and hooks, see the following page
 
 - <https://buildkite.com/docs/agent/v3/hooks>
 - <https://buildkite.com/docs/agent/v3/configuration>
+
+### Setup Pacman Package Cache
+
+<Alert type="info">
+
+Whenever we update our base archlinux image [here](https://github.com/NUbots/NUbots/blob/master/docker/Dockerfile#L13), both the mirror in the Dockerfile AND the mirror in the cache config need to be updated ([here](https://github.com/NUbots/NUbots/blob/master/docker/Dockerfile#L14) and [here](https://github.com/NUbots/NUbots/blob/master/doc/PacmanCache/docker-compose.yml#L14))
+
+</Alert>
+
+1. Create a directory to store cache config
+
+   ```
+   mkdir ~/pacman-cache && cd ~/pacman-cache
+   ```
+
+2. Download install script
+
+   ```
+   wget https://raw.githubusercontent.com/NUbots/NUbots/master/doc/PacmanCache/install.sh
```

Comment:
We should consider adding this script to [the agent setup script](https://github.com/NUbots/NUbook/blob/master/src/book/02-system/01-foundations/assets/ci_setup.sh).

Reply:
Probably a good idea, but also keep it as its separate script so people can set it up on their personal machines if they want.